q

```diff
@@ -0,0 +1,53 @@
+  authors_added[m
```

Comment:
Looks like this sneaked in here - please delete it.

Reply:
(I mean this whole file, in case that isn't clear)

Reply:
that is weird i would remove it right away!
Thats for spotting it.